src/components/reportPage/ReportPage.jsx

```diff
@@ -5,6 +5,9 @@ import Select from "react-select";
 import makeAnimated from "react-select/animated";
 
 import "./results-page.css";
+import ImageResultsInfoGuide from "./ImageResultsInfoGuide";
+
+import Repository from "../../data/Repository";
```

Comment:
This has been renamed to `LiveJobRepository.jsx`. That's why you're getting a build error